js/components/form/base-form.js

```diff
@@ -195,7 +195,15 @@ export default {
 
             Array.prototype.map.call(elements, function(el) {
                 let value;
-                if (TEXT_TAGS.includes(el.tagName.toLowerCase()) || TEXT_INPUTS.includes(el.type.toLowerCase())) {
+                if (el.tagName.toLowerCase() === 'select' && el.multiple) {
+                    value = []
+                    for (const index in el.options) {
```

Comment:
Actually, that is what I wanted to do.

But el.options is not really an arry, so I had to call Array.prototype.map.call and Array.prototype.filter.call, which made the code a bit hard to read.